build/gitversion.yml

```diff
@@ -0,0 +1,35 @@
+assembly-versioning-scheme: MajorMinorPatch
+mode: ContinuousDeployment
+next-version: 0.2.0
+continuous-delivery-fallback-tag: ""
+increment: none # Disabled as it is not used. Saves time on the GitVersion step
+branches:
+  master:
+    regex: master
+    tag: dev
+    increment: none
+  beta:
+    regex: release/beta/
+    tag: beta
+    increment: none
+    source-branches: [master]
+  stable:
+    regex: release/stable/
+    tag: ''
+    increment: none
+    source-branches: [master]
+  dev:
+    regex: dev/.*?/(.*?)
+    tag: dev.{BranchName}
+    source-branches: [master]
+  projects:
```

Comment:
we can remove the projects regex (from line 25 to 28). On Github it's not useful, it would mean we have a project name in the branch.  We would use Feature if we really needed.